crypto/ui/ui_util.c

```diff
@@ -47,8 +47,6 @@ int UI_UTIL_read_pw(char *buf, char *buff, int size, const char *prompt,
             ok = UI_process(ui);
         UI_free(ui);
     }
-    if (ok > 0)
-        ok = 0;
```

Comment:
Not sure about this change.  There was a reason for it: 364c7cd6216bc9b5ad35cd556b8381c88a0f22e7

Following up on that, PR 456 doesn't seem like #456, but might very well refer to a gitlab instance we had years ago that doesn't exist any more...  [sigh]

Reply:
Either way, this removal doesn't seem at all necessary for *this* PR, so please remove the removal

Reply:
Yeah, this hunk shouldn't actually matter in terms of fixing the issue. So @bernd-edlinger please revert it.

Reply:
@levitte you won. :grin:

Reply:
but it is unreachable code, the last statement before this reads:
```C
        if (ok >= 0)
            ok = UI_process(ui);
```
and when you look at UI_process you will see that it can only return
0, -1, or -2, while -2 is for a non-redoable error, that's why I chose -2
as a default, and that's why I think this statement is mis-leading.

Reply:
Hmmm...  yeahok